packages/dds/sequence/README.md

```diff
@@ -595,8 +595,8 @@ There are plans to make attribution policies more flexible, for example tracking
     -   [smde](https://github.com/microsoft/FluidFramework/tree/main/examples/data-objects/smde)
 
 -   Plain Text Editor Implementations
-    -   [collaborativeTextArea](https://github.com/microsoft/FluidFramework/blob/main/experimental/framework/react-inputs/src/CollaborativeTextArea.tsx)
-    -   [collaborativeInput](https://github.com/microsoft/FluidFramework/blob/main/experimental/framework/react-inputs/src/CollaborativeInput.tsx)
+    -   [collaborativeTextArea](https://github.com/microsoft/FluidFramework/blob/main/examples/utils/example-utils/src/reactInputs/CollaborativeTextArea.tsx)
```

Comment:
Should we just remove these entries, since this is a public facing doc?

Reply:
I think it's probably OK - the links above here also link to our internal examples (webflow, smde, etc.) and even though it's a public-facing doc the links will still function to get them to the repo to view the usage.  Would obv. prefer a more polished example to point to, but I think this maintains the status quo at least.